common-lib/src/main/scala/com/gu/mediaservice/model/Image.scala

```diff
@@ -29,6 +29,29 @@ case class Image(
   syndicationRights:   Option[SyndicationRights] = None
 ) {
   def rcsPublishDate: Option[DateTime] = syndicationRights.flatMap(_.published)
+
+  def syndicationStatus: SyndicationStatus = {
+    val isRightsAcquired: Boolean = syndicationRights.exists(_.isRightsAcquired)
+
+    if (!isRightsAcquired) {
+      UnsuitableForSyndication
+    } else {
+      val hasSyndicationUsage = usages.exists(_.platform == SyndicationUsage)
+
+      if (hasSyndicationUsage) {
+        SentForSyndication
+      } else {
+        val allowSyndicationLease = leases.leases.find(_.access == AllowSyndicationLease)
+        val denySyndicationLease = leases.leases.find(_.access == DenySyndicationLease)
+
```

Comment:
Isn't the case that an image can only have one syndication lease? If so, maybe we can simplify this a bit. 
e.g:
```
leases.leases.find(_.isSyndication).map(_.access) match {
    case Some(AllowSyndicationLease) => QueuedForSyndication
    case Some(DenySyndicationLease) => BlockedForSyndication
   case None => AwaitingReviewForSyndication
}
```

Reply:
Sadly, that rule's only enforced client-side atm 😞